### PR TITLE
[jQuery] Fix typos

### PR DESCRIPTION
**Pull request by stonio**
Fix basic typo in jQuery definition file.

### CODE REVIEW

File: jquery/index.d.ts [hunk 1]
```diff
@@ -1220,7 +1220,7 @@ interface JQueryStatic {
      *
      * @param value The value to search for.
      * @param array An array through which to search.
-     * @param fromIndex he index of the array at which to begin the search. The default is 0, which will search the whole array.
+     * @param fromIndex the index of the array at which to begin the search. The default is 0, which will search the whole array.
```
Comment (King2500) [inline jquery/index.d.ts]: better "The"

File: jquery/index.d.ts [hunk 2]
```diff
@@ -1270,7 +1270,7 @@ interface JQueryStatic {
     /**
      * Check to see if a DOM node is within an XML document (or is an XML document).
      *
-     * @param node he DOM node that will be checked to see if it's in an XML document.
+     * @param node the DOM node that will be checked to see if it's in an XML document.
```
Comment (King2500) [inline jquery/index.d.ts]: better "The"

**Comment by johnnyreilly** [comment]
THanks!

File: jquery/index.d.ts (hunk 1 above)
Comment (stonio) [inline jquery/index.d.ts]: @johnnyreilly : I wanted to apply this recommendation but this pull request has been merged too quickly...
See #13730

**Pull request merged**